### ISSUE TITLE
🐛 [firebase_auth] OTP not sent to IPHONE 11 MAX devices

### ISSUE THREAD

**Issue by randeepbagga** [FirebaseExtended/flutterfire#7090]
My current flutter app in production uses firebase_auth:0.15.3 and flutter:1.22.5.  Its uses phone authetication to log users into firebase.  recently users with IPHONE 11 PRO MAX devices stopped getting OTP back. This works fine on other IPHones just not Max. Interestingly, if I go to IPHONE 11 Pro MAx Setings-Notifications, I don't see my app there. Or if I got to Settings-> My App, I just see "sIRI & Search and Bckground app eferesh (disabled)". This is odd. I expect to see "Camera", "Notifications" , "Cellular" inside Settings-App. I understand I am on old version of flutter and plugins but app phone authentication works fine on all other iphones.

---

## Additional context

---

### Flutter doctor

Run `flutter doctor` and paste the output below:

<details><summary>Click To Expand</summary>

```
PASTE OUTPUT INSIDE HERE
```

</details>

---

### Flutter dependencies

Run `flutter pub deps -- --style=compact` and paste the output below:

<details><summary>Click To Expand</summary>

```
PASTE OUTPUT INSIDE HERE
```

</details>

---

**Comment by markusaksli-nc** [comment]
Hi @randeepbagga 
Is there any chance you could upgrade to the latest flutter stable and use the latest version of `firebase_auth`? There have been a lot of major changes, bug fixes, and overall improvements.
Thank you

**Comment by randeepbagga** [comment]
Hi- Thanks for responding. App is in production working totally fine on all phones except iphone pro max. I tried upgrding flutter and firebase auth. It gave me lot of errors. Basically i have to  o upgrade many plugins and change code in stripe payment gateway. This really is painful and will require development and testing of entire app. 
I noticed that my prod and development apns certificates were expired on firebase console. Does this impact firebase auth phone authentication? All other phones are working fine.
